web-marketplace/src/components/organisms/RegistryLayout/RegistryLayout.constants.ts

```diff
@@ -2,5 +2,7 @@ export const fullWidthRegExp: RegExp = /projects\/[a-z-]+/;
 export const TX_ERROR_MODAL_BUTTON = 'CLOSE WINDOW';
 export const KEPLR_CONNECT_BUTTON = 'connect wallet';
 export const KEPLR_LINK_TEXT = 'set up a Keplr wallet →';
+export const COOKIES_TOP_BANNER_ACCEPT_LABEL = 'accept';
+export const COOKIES_TOP_BANNER_REJECT_LABEL = 'Reject';
```

Comment:
Add translations

Reply:
It was already translated in another PR.